src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -122,7 +125,13 @@ public Loading(LoadingStyle loadingStyle) : base(loadingStyle)
         /// Get style of loading.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
-        public new LoadingStyle Style => ViewStyle as LoadingStyle;
+        public new LoadingStyle Style
```

Comment:
Hello,
is this for only getting style?
I think we need to add some comment and explanation how style can be set.
maybe style setting is only possible by using constructor and ApplyStyleXXX() method?

Reply:
Please add a comment that it returns copied Style instance, so that changing it does not effect to the corresponding View.

Reply:
OK. I will add